credits.html

```diff
@@ -20,114 +20,212 @@
   <link href="https://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
   <link rel="stylesheet" type="text/css" href="css/footer.css">
   <script type="text/javascript" src="js/materialize.min.js"></script>
-  <style type="text/css">
-    html {
-      box-sizing: border-box;
-    }
-
-    *,
-    *:before,
+  <script type="text/javascript" src="js/credits.js"></script>
+  <!-- Files for responsive nav bar -->
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/1.12.4/jquery.min.js" integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ=" crossorigin="anonymous"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
```

Comment:
Don't use External Linking. This was recommended by @niccokunzmann 

Read this : https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md#hosting-assets

Reply:
@duttaditya18 I need the bootstrap file because my full code depends on that. I have got the link from https://getbootstrap.com/ which is licensed.I don't see any problem with this.If u know some other site for bootstrap,Can you give me its link?

Reply:
You can download it from there: https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js

Reply:
We also have JQuery in the js folder.

Reply:
@niccokunzmann Okay i'll add it from there.Thanks!